bin/publish.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+mkdir -p public
+rsync -ric --exclude=/public \
+      --include=reveal.js/dist --include=reveal.js/plugin --exclude=reveal.js/* \
+      --exclude=/.* --include=* ./ public/
+sass --update reveal.js/css/print:public/reveal.js/dist/print
+sass -E utf-8 --update css public/css
```

Comment:
I realize it's just a helper script, but some error checking would probably still be worthwhile.

Reply:
Do you have anything special in mind?

I've seen sass fail quite a few times while I was hacking on the CSS. It produces more or less understandable output for the dev to fix the issue.

Reply:
nothing special - exit with non-zero if any of the commands fail?

Reply:
should be fixed with latest push